### ISSUE TITLE
[GwR] Collection manager

### ISSUE THREAD

**Issue by DiemBTran** [scientist-softserv/palni-palci#394]
# Summary

refs #386

Collection Managers should be able to create new collections as subcollection from the `Dashboard` > `Collection show` page. However, the `Create new collection as subcollection` link leads to `You are not authorized to access this page.`

# Acceptance Criteria

- Collection manager: can create new subcollections as subcollection
     * Can > Subcollections in Collection
          * [ ] Dashboard > Collection show > `Create new collection as subcollection` link
               * Bug: `You are not authorized to access this page.`

# Screenshots or Video

Attached video: Collection manager cannot create a new collection as a subcollection :x:. However, they can add an existing collection as a subcollection ✅ 

https://user-images.githubusercontent.com/29311858/212788671-94a9d422-05bc-4301-9488-aed12998b1b4.mp4

# Testing Instructions

- [ ] Test this on https://demo-tenant.hyku-demo.notch8.cloud/
- [ ] Add a non-admin user to a group that only has the role: collection manager. From `Dashboard` > `Manage Users`, make sure that non-admin user has no other roles besides collection manager.
- [ ] As the non-admin user, navigate to the `Dashboard` > `Collection show page` for a collection you do not own.
- [ ] Click the `Create new collection as subcollection` link and attempt to create a subcollection

**Comment by bkiahstroud** [comment]
This will indirectly be resolved by https://github.com/scientist-softserv/palni-palci/issues/398.

TL;DR -- A solr migration on the server should resolve this bug; no code changes are necessary.  

The solr schema used on https://hyku-demo.notch8.cloud/ is missing a required migration step for all groups w/ roles permissions to work properly. This bug is due to that migration step not being run yet. https://github.com/scientist-softserv/palni-palci/issues/398 will add instructions on how to complete this migration step, which will be enough to unblock this for all Hyku adopters who upgrade. 

As for https://hyku-demo.notch8.cloud/ specifically, we will need to run this same migration step once it's documented.

**Comment by bkiahstroud** [comment]
This will be fixed by https://github.com/scientist-softserv/palni-palci/issues/412
